{"uid":"62d3b81cf0b4f89b","name":"test_deploy_async_replication_setup","fullName":"tests.integration.ha_tests.test_async_replication#test_deploy_async_replication_setup","historyId":"d78aa2d7506528b71f817b6a92e55938","time":{"start":1722130531191,"stop":1722130779207,"duration":248016},"description":"Build and deploy two PostgreSQL cluster in two separate models to test async replication.","descriptionHtml":"<p>Build and deploy two PostgreSQL cluster in two separate models to test async replication.</p>\n","status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: postgresql/2","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/tests/integration/ha_tests/test_async_replication.py\", line 139, in test_deploy_async_replication_setup\n    await gather(\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2963, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2869, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuUnitError: Unit in error: postgresql/2","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1722130530124,"stop":1722130530125,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1722130530125,"stop":1722130530343,"duration":218},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1722130530125,"stop":1722130530125,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"first_model","time":{"start":1722130530343,"stop":1722130530343,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"second_model","time":{"start":1722130530344,"stop":1722130531188,"duration":844},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm","time":{"start":1722130531189,"stop":1722130531191,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1722130531191,"stop":1722130531191,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Build and deploy two PostgreSQL cluster in two separate models to test async replication.","status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: postgresql/2","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/tests/integration/ha_tests/test_async_replication.py\", line 139, in test_deploy_async_replication_setup\n    await gather(\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2963, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/runner/actions-runner/_work/test-runners-2-azure-arm64-postgresql-operator/test-runners-2-azure-arm64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2869, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuUnitError: Unit in error: postgresql/2","steps":[],"attachments":[{"uid":"7a7c76ab6486b9b","name":"log","source":"7a7c76ab6486b9b.txt","type":"text/plain","size":5406}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1722130779232,"stop":1722130779232,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm::finalizer","time":{"start":1722130779307,"stop":1722130779307,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"second_model::finalizer","time":{"start":1722130779308,"stop":1722130779431,"duration":123},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1722130779431,"stop":1722130781118,"duration":1687},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1722130781119,"stop":1722130781119,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1722130781119,"stop":1722130781119,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1722130781119,"stop":1722130781119,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"@pytest.mark.skipif(False, reason='Requires juju 3')"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_async_replication"},{"name":"host","value":"runner"},{"name":"thread","value":"9155-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_async_replication"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"439f248cf001b20","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":10,"unknown":3,"total":14},"items":[{"uid":"8b281f8c840cf8f5","reportUrl":"../88//#testresult/8b281f8c840cf8f5","status":"passed","time":{"start":1722044171072,"stop":1722044825731,"duration":654659}},{"uid":"344921a0f56922f7","reportUrl":"../87//#testresult/344921a0f56922f7","status":"passed","time":{"start":1721957719368,"stop":1721958391871,"duration":672503}},{"uid":"57b4a55bca6ad9cb","reportUrl":"../86//#testresult/57b4a55bca6ad9cb","status":"passed","time":{"start":1721871375174,"stop":1721872023510,"duration":648336}},{"uid":"cf86b36e0264edf5","reportUrl":"../85//#testresult/cf86b36e0264edf5","status":"passed","time":{"start":1721785266006,"stop":1721785911305,"duration":645299}},{"uid":"d86b2d47c26e4485","reportUrl":"../84//#testresult/d86b2d47c26e4485","status":"passed","time":{"start":1721698448183,"stop":1721699084155,"duration":635972}},{"uid":"190a65eb823ec341","reportUrl":"../83//#testresult/190a65eb823ec341","status":"passed","time":{"start":1721612477976,"stop":1721613109978,"duration":632002}},{"uid":"6413e34c8916d59b","reportUrl":"../81//#testresult/6413e34c8916d59b","status":"passed","time":{"start":1721440253622,"stop":1721440894698,"duration":641076}},{"uid":"5695d9842c667c8e","reportUrl":"../78//#testresult/5695d9842c667c8e","status":"passed","time":{"start":1721356856735,"stop":1721357519121,"duration":662386}},{"uid":"cbf815bc38ef0324","reportUrl":"../75//#testresult/cbf815bc38ef0324","status":"passed","time":{"start":1718156528175,"stop":1718157573251,"duration":1045076}},{"uid":"159fed9294c12667","reportUrl":"../73//#testresult/159fed9294c12667","status":"unknown","time":{}},{"uid":"34fa4f107d2471a6","reportUrl":"../72//#testresult/34fa4f107d2471a6","status":"unknown","time":{}},{"uid":"89d80102cb82c598","reportUrl":"../71//#testresult/89d80102cb82c598","status":"unknown","time":{}},{"uid":"6c870be3b79e4a70","reportUrl":"../70//#testresult/6c870be3b79e4a70","status":"passed","time":{"start":1717811964427,"stop":1717813028981,"duration":1064554}}]},"tags":["asyncio","abort_on_fail","@pytest.mark.skipif(False, reason='Requires juju 3')","group(1)"]},"source":"62d3b81cf0b4f89b.json","parameterValues":[]}